tensorflow/python/estimator/inputs/queues/feeding_functions.py

```diff
@@ -320,7 +320,7 @@ def _enqueue_data(data,
       ]
       queue_shapes = [()] + [col.shape[1:] for col in data.values()]
       get_feed_fn = _OrderedDictNumpyFeedFn
-    elif isinstance(data, tp.FunctionType):
```

Comment:
if you look at lines 314,318 above types is being used as a variable and will overwrite the import that's why I imported it as tp